src/js/index.spec.js

```diff
@@ -1,4 +1,7 @@
-import './index.js';
+/**
```

Comment:
Do you need `jsdom` loaded in this placeholder test file?

Reply:
Ah, I see what you were talking about @changangus. This issue is because JDOM will mock out basic DOM elements, but not the `addEventListener` method on elements.

One approach you could take would be to do a small wrapper implementation around the `addEventListener` API that would allow you to mock out the contents of the wrapper (`jest.mock('./path/to/addEventListener/mock`) while testing the rest of the code. This, I think, would be the easiest way forward.

Another approach would be to _completely_ mock out the `eventHandlers.js` module with something like (mock module)[https://jestjs.io/docs/mock-functions#mocking-modules] or (mock implementation)[https://jestjs.io/docs/mock-functions#mock-implementations], but that would mean that tests on index.js wouldn't _actually_ be importing the real version of `eventHandlers`, which would open up the possibility of passing tests when the application code fails.

Lastly, another approach would include globally mocking out the `addEventListener` to mocked out elements in the DOM, but that would require mocking them out in _all_ tests that import the `eventHandlers` file which would get tedious and messy relatively quickly.